package.json

```diff
@@ -51,7 +61,7 @@
     "@ethersproject/bignumber": "5",
     "@ethersproject/contracts": "5",
     "@reduxjs/toolkit": "1",
-    "react": "17",
+    "react": "^16.14.0",
```

Comment:
what makes these necessary?

Reply:
you're right its not, accidentally changed during my testing experimentations. reverted